projects/plugins/jetpack/tests/php/sync/test_class.jetpack-sync-themes.php

```diff
@@ -250,8 +250,8 @@ private function perform_network_enable_disable_assertions( $test_themes, $event
 	}
 
 	public function test_install_edit_delete_theme_sync() {
-		$theme_slug = 'itek';
-		$theme_name = 'iTek';
+		$theme_slug = 'astra';
```

Comment:
> I'm not sure how to search for similar themes directly, but it seems any theme with the "Community" badge in the WordPress.com theme search would work:

Maybe worth a comment here pointing that out?

Reply:
ec4fe1b835